README.adoc

```diff
@@ -6,135 +6,308 @@
 :icons: font
 :source-highlighter: prettify
 
-This guide walks you through the process of consuming a link:/understanding/SOAP[SOAP-based web service] with Spring.
+This guide walks you through the process of consuming a SOAP-based web service with
```

Comment:
You don't like the "understanding" links? I guess there's a Wikipedia link right underneath, so probably redundant?